### PR TITLE
Add perf test for multi-key maps.

### PR DESCRIPTION
**Pull request by willhite**

### CODE REVIEW

File: samples/go/csv/csv-import/perf_test.go [hunk 1]
```diff
@@ -39,6 +39,24 @@ func (s *perfSuite) SetupSuite() {
 	assert.NoError(s.T, err)
 }
 
+func (s *perfSuite) Test001ImportChiBuildingViolationsFromBlobAsMultiKeyMap() {
```
Comment (kalman) [inline samples/go/csv/csv-import/perf_test.go]: you're going to add 1000 tests? I think 01 (and likewise 02) is sufficient :-)

**Comment by mikegray** [comment]
lgtm

File: samples/go/csv/csv-import/perf_test.go [hunk 2]
```diff
@@ -39,6 +39,24 @@ func (s *perfSuite) SetupSuite() {
...
+
+	blob := types.NewBlob(io.MultiReader(files...))
+	fmt.Fprintf(s.W, "\tchi-building-violations is %s\n", humanize.Bytes(blob.Len()))
+
+	ds := dataset.NewDataset(s.Database, "chi-building-violations/raw")
+	_, err := ds.CommitValue(blob)
+	assert.NoError(err)
+}
+
+func (s *perfSuite) Test002ImportChiBuildingViolationsFromBlobAsMultiKeyMap() {
```
Comment (willhite) [inline samples/go/csv/csv-import/perf_test.go]: Oops. These end up with the same name in the perf db, I'll change that.

**Comment by kalman** [comment]
jenkins: perf this

**Pull request merged**